model/span.go

```diff
@@ -0,0 +1,49 @@
+package model
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// SpanContext holds the context of a Span.
+type SpanContext struct {
+	TraceID  TraceID `json:"traceId"`
+	ID       ID      `json:"id"`
+	ParentID *ID     `json:"parentId,omitempty"`
+	Debug    bool    `json:"debug,omitempty"`
+	Sampled  *bool   `json:"-"`
+	Err      error   `json:"-"`
+}
+
+// SpanModel structure.
+//
+// If using this library to instrument your application you will not need to
+// directly access or modify this representation. The SpanModel is exported for
+// use cases involving 3rd party Go instrumentation libraries desiring to
+// export data to a Zipkin server using the Zipkin V2 Span model.
+type SpanModel struct {
+	SpanContext
+	Name           string            `json:"name"`
+	Kind           Kind              `json:"kind,omitempty"`
+	Timestamp      time.Time         `json:"timestamp,omitempty"`
+	Duration       time.Duration     `json:"duration,omitempty"`
```

Comment:
I believe Timestamp and Duration could be pointers, they are empty before the start.

Reply:
pointers just complicate things if not needing to signal distinction between zero value and empty